packages/cli/src/databases/migrations/common/1717498465932-RefactorExecutionIndices.ts

```diff
@@ -0,0 +1,46 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+export class RefactorExecutionIndices1717498465932 implements ReversibleMigration {
+	async up({ schemaBuilder, isPostgres, isSqlite }: MigrationContext) {
+		await schemaBuilder.dropIndex(
+			'execution_entity',
+			['waitTill', 'id'], // covered by `waitTill, status` - `id` is unused
+			{
+				customIndexName: isPostgres
+					? 'IDX_85b981df7b444f905f8bf50747'
+					: 'IDX_b94b45ce2c73ce46c54f20b5f9',
+				skipIfMissing: true,
+			},
+		);
+
+		if (isSqlite) {
+			await schemaBuilder.dropIndex('execution_entity', ['stoppedAt'], {
+				customIndexName: 'idx_execution_entity_stopped_at', // duplicate of `IDX_execution_entity_stoppedAt`
```

Comment:
We can probably remove both right? Not just the duplicate but both, as these are not used by any query